### PR TITLE
Feature/46314 pcs outbound transfer restrict to household only

### PR DESCRIPTION
**Pull request by GuidoWorkDev**
added integration test WhenIGetEvidenceNotesBasedSameComplianceYear
to verify only notes with same compliance year are expected - notes with CY-1 and CY+1 not exoected of being returned.

### CODE REVIEW

File: src/EA.Weee.Web/Areas/Scheme/Controllers/ManageEvidenceNotesController.cs [hunk 1]
```diff
@@ -84,7 +84,7 @@ public ActionResult Transfer(Guid organisationId)
                 var result = await client.SendAsync(User.GetAccessToken(),
                 new GetEvidenceNotesByOrganisationRequest(organisationId,
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Scheme/Controllers/ManageEvidenceNotesController.cs:85]: Revert this change, we dont need this filter on normal evidence notes

File: src/EA.Weee.DataAccess/DataAccess/EvidenceDataAccess.cs [hunk 2]
```diff
@@ -118,6 +118,7 @@ public async Task<IEnumerable<Note>> GetNotesToTransfer(Guid schemeId, List<int>
                 .IncludeFilter(n => n.NoteTonnage.Select(nt => nt.NoteTransferTonnage).Select(ntt => ntt.Select(nttt => nttt.TransferNote)))
                 .Where(n => n.RecipientId == schemeId &&
                     n.NoteType.Value == NoteType.EvidenceNote.Value &&
+                    n.WasteType.Value == WasteType.HouseHold &&
```
[the remark this answers is not in the document]
Comment (GuidoWorkDev) [reply src/EA.Weee.DataAccess/DataAccess/EvidenceDataAccess.cs:121]: modified a couple of integration tests to verify WatseType = household on returned notes
